### PR TITLE
Add basis for rbac models

### PR DESCRIPTION
**Pull request by Convly**
## Description of what you did:

Add `admin::User` & `admin::Role` models for RBAC.

### CODE REVIEW

File: packages/strapi-admin/models/Role.settings.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "collectionName": "strapi_role",
```
Comment (derrickmehaffy) [inline packages/strapi-admin/models/Role.settings.json:2]: I'm not entirely sold on this collection name, is there a standard that we should be applying here?

File: packages/strapi-admin/models/User.settings.json [hunk 2]
```diff
@@ -1,16 +1,30 @@
 {
   "collectionName": "strapi_administrator",
```
Comment (derrickmehaffy) [inline packages/strapi-admin/models/User.settings.json:2]: While I know this one is a tad older coming from the beta, I'm wondering if we should have an RFC to describe how plugins and packages name their collections by default. With the `users-permissions` you can clearly identify which plugin the collection/table belongs to.

Comment (alexandrebodin) [reply packages/strapi-admin/models/User.settings.json:2]: That's an init PR. We will certainly change the names later during the dev

Comment (derrickmehaffy) [reply packages/strapi-admin/models/User.settings.json:2]: Alright cool, it just seemed odd to start the collection name with `strapi_` (not sure why this was chosen for `strapi-admin` anyway)

File: packages/strapi-admin/models/Role.settings.json (hunk 1 above)
Comment (Convly) [reply packages/strapi-admin/models/Role.settings.json:2]: As @alexandrebodin replied in the other thread, this is the first iteration that we need to begin the development of other related features.
This name will most likely be changed in the future.

File: packages/strapi-utils/lib/models.js [hunk 3]
```diff
@@ -327,15 +327,8 @@ module.exports = {
...
-        if (association.plugin) {
-          details = _.get(
-            strapi.plugins,
-            [association.plugin, 'models', targetName, 'attributes', association.via],
-            {}
-          );
-        } else {
-          details = _.get(strapi.models, [targetName, 'attributes', association.via], {});
-        }
+        const models = strapi.db.getModelsByPluginName(association.plugin);
```
Comment (alexandrebodin) [inline packages/strapi-utils/lib/models.js:330]: why not use directly `strapi.db.getModel(targetName, association.plugin)` ?

Comment (Convly) [reply packages/strapi-utils/lib/models.js]: No specific reason, good catch!

**Pull request merged**